### PR TITLE
[DOCS] revert getting started tutorial to RBP process

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
- copies the data assistant version of the create expectations tutorial page to the docs_pending folder (so it can be restored once the Onboarding Assistant and Data Assistant docs are in place/APIs are stable)
- reverts the guidance on the "Create Expectations" tutorial page to give instructions for working with a RBP. (Leaves the updates on the "Connect to data" tutorial page in place; they should reflect the updated code to remove the prior deprecation warnings).

After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [X] I have made corresponding changes to the documentation

Thank you for submitting!

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=5307&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=5307&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=5307&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map Legend

**Review by kenwade4 — APPROVED** [review]

**Review by rdodev — APPROVED** [review]

File: docs/tutorials/getting_started/tutorial_create_expectations.md [hunk 1]
```diff
@@ -101,9 +101,9 @@ Since notebooks are often less permanent, creating Expectations in a notebook al
 ```
 
 **Cell 3**
-![Cell3](../../images/getting_started_tutorial/tutorial_create_expectations_cell3_onboarding_assistant.png)
+![Cell3](../../images/getting_started_tutorial/tutorial_create_expectations_cell3.png)
```
Comment (anthonyburdi) [inline docs/tutorials/getting_started/tutorial_create_expectations.md:104]: @Rachel-Reverie I just noticed a few items to change in this image, see [my friction log](https://superconductive.atlassian.net/wiki/spaces/DVRL/pages/418906113/Getting+Started+Tutorial+-+Friction+Log+20220614+10+11EST) for more details.

**Review by anthonyburdi — CHANGES_REQUESTED** [review]
Just a few small changes requested to the image

**Review by anthonyburdi — APPROVED** [review]
LGTM!

File: docs/tutorials/getting_started/tutorial_create_expectations.md (hunk 1 above)
Comment (Rachel-Reverie) [reply docs/tutorials/getting_started/tutorial_create_expectations.md:104]: Fixed with update to png for that cell.

**Pull request merged**